pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -449,13 +468,18 @@ func TestPlugin(t *testing.T) {
 				nodes = []*v1.Node{workerNode}
 			}
 			testCtx := setup(t, nodes, tc.claims, tc.classes, tc.schedulings)
+			testCtx.p.enabled = !tc.disable
 
 			initialObjects := testCtx.listAll(t)
 			result, status := testCtx.p.PreFilter(testCtx.ctx, testCtx.state, tc.pod)
 			t.Run("prefilter", func(t *testing.T) {
 				assert.Equal(t, tc.want.preFilterResult, result)
 				testCtx.verify(t, tc.want.prefilter, initialObjects, result, status)
 			})
+			// When the plugin is disabled, only the PreFilter is currently tested
+			if tc.disable {
```

Comment:
Instead of having `disable` field on tc, how about just determining whether run `Filter` or not via returning status?

```
if status.IsSkip() {
  return
}
```

Reply:
PTAL, thanks.